masonry/src/properties/border_color.rs

```diff
@@ -13,17 +13,22 @@ pub struct BorderColor {
     pub color: AlphaColor<Srgb>,
 }
 
-impl Property for BorderColor {}
+impl Property for BorderColor {
+    fn static_default() -> &'static Self {
+        static DEFAULT: BorderColor = BorderColor {
+            color: AlphaColor::TRANSPARENT,
+        };
+        &DEFAULT
+    }
+}
 
 // TODO - The default border color in CSS is `currentcolor`,
 // the color text is displayed in.
 // Do we want to implement that?
 
 impl Default for BorderColor {
     fn default() -> Self {
-        Self {
-            color: AlphaColor::from_rgba8(0, 0, 0, 0),
-        }
+        *Self::static_default()
```

Comment:
Why's this different from the one for background? (I mean different in terms of a clone vs a `*`.)

Reply:
Background doesn't implement Copy.